templates/Gemfile.erb

```diff
@@ -37,7 +37,8 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "pry-byebug"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.4.0"
+  gem "refills"
+  gem "rspec-rails", "~> 3.5.0.beta4"
```

Comment:
Should we merge even when `rspec-rails` is still in beta?  https://github.com/rspec/rspec-rails/releases